src/motile_plugin/data_views/views_coordinator/collections.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from PyQt5.QtCore import QObject
+
+
+class Collection(QObject):
```

Comment:
I'm not sure why we need this instead of just having a normal list. Perhaps having it be a QObject is necessary?